src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorUnlockVerify.ts

```diff
@@ -38,25 +49,133 @@ export const transformSelectAuthenticatorUnlockVerify: IdxStepTransformer = ({
     stepName,
   );
 
-  const title: TitleElement = {
+  const identifierContainer: IdentifierContainerElement = {
+    type: 'IdentifierContainer',
+    options: {
+      identifier: '',
+    },
+  };
+
+  const unlockAccountTitle: TitleElement = {
     type: 'Title',
     options: {
       content: loc('unlockaccount', 'login'),
     },
   };
 
+  const verifyTitle: TitleElement = {
+    type: 'Title',
+    options: {
+      content: loc('oie.select.authenticators.verify.title', 'login'),
+    },
+  };
+
+  const verifySubtitle: DescriptionElement = {
+    type: 'Description',
+    contentType: 'subtitle',
+    options: {
+      content: loc('oie.select.authenticators.verify.subtitle', 'login'),
+    },
+  };
+
   const identifier = getUIElementWithName('identifier', uischema.elements) as FieldElement;
 
-  const authenticatorListElement: AuthenticatorButtonListElement = {
+  if (identifier) {
+    // add username/identifier from config if provided
+    if (username) {
+      data.identifier = username;
+    } else if (typeof identifier.options.inputMeta.value === 'string') {
+      data.identifier = identifier.options.inputMeta.value;
+    // TODO: OKTA-508744 - to use rememberMe in features once Default values are added widgetProps.
+    // (i.e. rememberMe is default = true in v2)
+    } else if (typeof features?.rememberMe === 'undefined' || features?.rememberMe === true) {
+      const usernameCookie = getUsernameCookie();
+      data.identifier = usernameCookie;
+    }
+  }
+
+  const authenticatorList: AuthenticatorButtonListElement = {
     type: 'AuthenticatorButtonList',
     options: { buttons: authenticatorButtons, dataSe: 'authenticator-verify-list' },
   };
 
-  uischema.elements = [
-    title,
+  const identifyWithUsernameLayout: UISchemaLayout = {
+    type: UISchemaLayoutType.VERTICAL,
+    elements: [],
+  };
+
+  // TODO: Refactor transformer once we have backend support for this Stepper flow - OKTA-657627
+  const nextButton: StepperButtonElement = {
+    type: 'StepperButton',
+    label: loc('oform.next', 'login'),
+    options: {
+      type: ButtonType.SUBMIT,
+      variant: 'primary',
+      nextStepIndex: (widgetContext) => {
+        const { setMessage, data: updatedData } = widgetContext;
+
+        if (updatedData.identifier && typeof updatedData.identifier === 'string') {
+          setMessage(undefined);
```

Comment:
Can this be moved outside of the `if` check (effectively always clearing the message whenever they click next and having the below logic re-add it if necessary)?